cluster/bin/snappy-job.sh

```diff
@@ -119,6 +120,15 @@ while (( "$#" )); do
       shift
       contextName="${1:-$TOK_EMPTY}"
     ;;
+    --user)
+      shift
+      username="${1:-$TOK_EMPTY}"
+      while [ -z ${password} ]; do
+        read -s -p "Enter password for ${username}: " password
+        echo ""
+      done
+      securePart="-u ${username}:${password}"
+    ;;
```

Comment:
@ashetkar Even if curl somehow does hide the arguments (not sure if it can alter an OS "feature" but perhaps there is some kernel API to do that), this shell script itself will not and will be seen in the script arguments. Using the "--netrc" option is the proper way.